Update `Official beatmapping contest support` (#12921)
ppy/osu-wiki: wiki/Contests/Official_support/en.md

### Contests

Community-run contests receiving this support are expected to abide by the following general ideals:

- Remain free and fair.
- Be as transparent as possible.
- Preserve competitive integrity.
- Encourage good sportsmanship.

In addition, all promotional material or any services associated with a contest receiving official support should adhere to the [osu! community rules](/wiki/Rules). This includes things like Twitter accounts, Discord servers, and so on.

### Additional expectations

While the following expectations mainly apply to tournaments, contests are still expected to abide by them to the best of their ability:

- [Staff](/wiki/Tournaments/Official_support#staff)
- [Players](/wiki/Tournaments/Official_support#players)

## Eligibility

Contests which abide by the following criteria are eligible for official support for **profile badges**:

- **The contest must meet either of these criteria:**
  - The contest is organised and run by an experienced team of accomplished and reputable staff, as decided by the osu! team.
  - The contest has successfully run for one iteration without official support.

> - **The forum threads associated with the contest MUST contain a clearly visible and accessible link to the contest reports form.** The link must be placed at the bottom of the original post.

Reviewer: ```suggestion
- **The forum threads associated with the contest MUST contain a clearly visible and accessible link to the contest report form.** The link must be placed at the bottom of the original post.
```

Reviewer: for some words plural sounds more natural inside compounds (like results), but i can't find "reports form" used anywhere else on the internet. generally, words inside compound terms aren't pluralised

Author: "The form for contest reports" rather than "The form to report a contest" that's how I read the difference between the one and another.

In any case, if it's changed here, may as well change it in the other. At the end of the day it's just 1 (one) letter, so I do not mind it either way.

Reviewer 2: "report form" flows more naturally imo

- **The contest must have a clearly defined judging criteria based on mapping merit.** These ensure a contest follows an understandable quality standard in line with other mapping contests.
- **The contest's results must be determined by valuing scores from all judges equally.** Using standard deviation to determine final scores is required.[^exception-note]
  - The final score must be calculated as follows: `judge X's final score = (judge X's raw score - judge X's average raw score) / judge X's standard deviation`.

> - **The contest's host must moderate their judges' comments.** Comments which are either offensive, belittling, low effort, or otherwise inappropriate may be removed at the committee's discretion.

Reviewer: grammatical nit

```suggestion
- **The contest's host must moderate their judges' comments.** Comments which are either offensive, belittling, of low effort, or otherwise inappropriate may be removed at the committee's discretion.
```

  - The final score must be calculated as follows: `judge X's final score = (judge X's raw score - judge X's average raw score) / judge X's standard deviation`.
- **The contest's host must moderate their judges' comments.** Comments which are either offensive, belittling, low effort, or otherwise inappropriate may be removed at the committee's discretion.
- **The contest's results must be publicly displayed with judge scores/comments visible.** The judges may be anonymised publicly, but the osu! team may ask for un-anonymised judge details if necessary.

> - **The contest must have at least 4 judges.** Contests with fewer judges allow each judge too much influence on overall results. This also applies to contests which incorporate a screening phase prior to judging.

Reviewer 2: wording feels pretty ehh here, this reads as we're also requiring 4 judges for contests which incorporate a screening phase prior to judging, which doesn't make sense as that's already the case.

I'd want us to use something that explicitly requires 4 or more screeners, i.e.

```suggestion
- **The contest must have at least 4 judges.** Contests with fewer judges allow each judge too much influence on overall results.
- **contests which incorporate a screening phase prior to judging must also have at least 4 screeners.**
```

Author: I agree. Was probably too stubborn not making a new bulletpoint altogether. An extra one won't hurt, will apply.
